### PR TITLE
docs: worked over weblate inconsistencies

### PR DESCRIPTION
**Pull request by smoe**
Went through the many "marked for edit"-entries in Weblate. Most of those Weblate changes were reverse-translations that have now been confirmed in their assignment to the translation. But for a good part there were unexpected
differences between English and the translation that required changes on either side.

### CODE REVIEW

File: docs/src/config/pncconf.adoc [hunk 1]
```diff
@@ -36,17 +36,17 @@ details.
 
 == Step by Step Instructions
 
-.PnCConf Splash
-image::images/pncconf-splash.png["PnCConf Configuration Wizard"]
+.PnCconf Splash
```
Comment (hansu) [inline docs/src/config/pncconf.adoc:39]: Does anyone has an idea for what "PnC" stands for? I guess  @jepler can answer this...

Comment (jepler) [reply docs/src/config/pncconf.adoc:39]: > Does anyone has an idea for what "PnC" stands for? I guess  @jepler can answer this...

I didn't originate this part of linuxcnc

Comment (hansu) [reply docs/src/config/pncconf.adoc:39]: As it is based on Stepconf, I assumed you originated this as well. My bad. A look into the sources gives me @c-morley as possible candidate for this question :slightly_smiling_face:

Comment (c-morley) [reply docs/src/config/pncconf.adoc:39]: I'm guilty (of course hugely based on Jeff's way-better-coded Stepconf at the time)
Point aNd Click

Comment (hansu) [reply docs/src/config/pncconf.adoc:39]: Thanks, great to have this clarified :slightly_smiling_face:

**Pull request merged**